docs/ReactVaporStore.ts

```diff
@@ -1,6 +1,20 @@
 import * as Redux from 'redux';
+// uncomment the line below for the redux logs to show in your chrome console
+// import {createLogger} from 'redux-logger';
 import thunk from 'redux-thunk';
 import {IReactVaporExampleState, Reducers} from './Reducers';
 
+if ((window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) {
+    (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({name: 'React Vapor'});
+}
+
 export const ReactVaporStore: Redux.Store<IReactVaporExampleState> =
-    Redux.createStore<IReactVaporExampleState>(Reducers, (window as any).devToolsExtension && (window as any).devToolsExtension.call(), Redux.applyMiddleware(thunk));
+    Redux.createStore<IReactVaporExampleState>(
+        Reducers,
+        (window as any).devToolsExtension && (window as any).devToolsExtension.call(),
+        Redux.applyMiddleware(
+            thunk,
+            // uncomment the line below for the redux logs to show in your chrome console
+            // createLogger(),

```

Comment:
I think we can use the `process.env.NODE_ENV` and only add the createLogger in `development` (like we do in UA)

Reply:
I actually commented it because it crates lots of logs and it seems to make the UI slower...which can be annoying/useful depending on your context

Reply:
As someone who always goes and disable it, commenting it by default is welcomed 👍